app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -84,8 +84,7 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
         request.targetUrl = 'https://ci.chromium.org/p/flutter/g/engine/console';
         request.context = 'luci-engine';
         if (status != GithubBuildStatusUpdate.statusSuccess) {
-          request.description = 'Flutter build is currently broken. Please do not merge this '
-              'PR unless it contains a fix to the broken build.';
+          request.description = config.flutterBuildDescription;
```

Comment:
Shall we say `Engine build`? `Flutter build` seems confusing as used for engine tree.

Reply:
What about `Tree is currently broken. ... contains a fix for the tree.`? That way it works for all Flutter repos?

Reply:
SGTM.